content/socials.js

```diff
@@ -24,7 +24,7 @@ export default class Socials {
     const { all } = Socials;
 
     let match = null;
-    const social = Object.values(all).find((social) => {
+    const social = Object.keys(all).map(key => all[key]).find((social) => {
```

Comment:
Ah, I thought Babel was taking care of this... Should have a proper Webpack config I guess